### PR TITLE
use typing.Literal for enum schemas

### PR DESCRIPTION
**Pull request by duncathan**
also slightly cleans up the output format

### CODE REVIEW

File: src/jsonschema_to_typeddict/converter.py [hunk 1]
```diff
@@ -127,6 +136,12 @@ def _convert_object_entry(entry_name: str, entry_value: dict) -> CodeResult:
     return CodeResult(block_result, type_name)
 
 
+def _convert_enum_entry(entry_name: str, entry_value: dict) -> CodeResult:
+    values = [repr(value) for value in entry_value["enum"]]
+    inline = _snake_case_to_pascal_case(entry_name)
+    return CodeResult(f"{inline} = typ.Literal[\n    {',\n    '.join(values)}\n]\n", inline)
```
Comment (henriquegemignani) [inline src/jsonschema_to_typeddict/converter.py:142]: Broke Python < 3.12

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

File: src/jsonschema_to_typeddict/converter.py (hunk 1 above)
Comment (duncathan) [reply src/jsonschema_to_typeddict/converter.py:142]: ugh, are we not using typing extensions? damn

Comment (henriquegemignani) [reply src/jsonschema_to_typeddict/converter.py:142]: It's actually an issue with the f-string

Comment (duncathan) [reply src/jsonschema_to_typeddict/converter.py:142]: > It's actually an issue with the f-string

oh LMAO ok that's easily fixed phew

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**